src/test/java/products/CreateProducts.java

```diff
@@ -0,0 +1,54 @@
+package products;
+import common.BaseURL;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.response.ResponseBody;
+import io.restassured.specification.RequestSpecification;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.util.HashMap;
+
+import static io.restassured.RestAssured.get;
+import static io.restassured.RestAssured.given;
+
+
+
+public class CreateProducts {
+
+    @Test
+    public void createProduct() {
+        BaseURL.setBaseUrl();
+
+        RequestSpecification request = given();
+
+        request.header("Content-Type", "application/json");
```

Comment:
naredni korak je da sve što imamo kao String ovako zakucano, a želimo da koristimo na više mesta, da ga izvučemo u zaseban fajl kao konstantu